go-controller/pkg/clustermanager/clustermanager.go

```diff
@@ -142,6 +145,10 @@ func NewClusterManager(ovnClient *util.OVNClusterManagerClientset, wf *factory.W
 		}
 	}
 
+	if config.OVNKubernetesFeature.EnableRouteAdvertisements {
+		cm.raController = *routeadvertisements.NewController(wf, ovnClient)
```

Comment:
Does it mean the BGP can only work in IC mode?

Reply:
Why do you ask in the context of this code/PR?

It will only work in BGP mode but that is because we plan to be able to import routes to OVN in IC mode only (due to time constraints) but there is no PR for that yet.